heat-stack/model/NaturalGasBillRecords.tsx

```diff
@@ -0,0 +1,7 @@
+export interface NaturalGasBillRecords {
+	provider: number
+	periodStartDate: Date
+	periodEndDate: Date
+	usageTherms: number
+	inclusionOverride: number
```

Comment:
`suggestion`: the database won't have a concept of it, but I might use an [enum ](https://www.typescriptlang.org/docs/handbook/enums.html) here.

Reply:
what should the options be?

Reply:
The rules engine models it like so: https://github.com/codeforboston/home-energy-analysis-tool/blob/main/rules-engine/src/rules_engine/pydantic_models.py#L13-L16

```py
class AnalysisType(Enum):
    DO_NOT_INCLUDE = 0
    INCLUDE = 1
    INCLUDE_IN_OTHER_ANALYSIS = -1
```

I'd use those names for consistency (in whatever the right casing is for Javascript).

Reply:
updated to `inclusionOverride: 'Include' | 'Do not include' | 'Include in other analysis'`